types/underscore/index.d.ts

```diff
@@ -6138,6 +6217,18 @@ declare namespace _ {
          */
         compose(...functions: Function[]): _Chain<T>;
 
+        /**
+         * Wrapped type `(...args: any[]) => any`.
+         * @see _.restArguments
+         */
+        restArguments(func:Function, startIndex:number): _Chain<T>;
+
+        /**
+         * Wrapped type `Array<number | string>`.
+         * @see _.toPath
+         */
+        toPath(path: number | string | Array<number | string>): _Chain<T>;
```

Comment:
The first argument is implicit in the Underscore and _Chain interfaces.

第一个参数隐含在“Underscore”和“_Chain”接口中。

```suggestion
        /**
         * Wrapped type `(...args: any[]) => any`.
         * @see _.restArguments
         */
        restArguments(startIndex:number): _Chain<T>;

        /**
         * Wrapped type `number | string | Array<number | string>`.
         * @see _.toPath
         */
        toPath(): _Chain<T>;
```